data/xml/Q20.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
2020 Should be done in the new format: collection id = "2020.tacl", and papers named "2020.tacl-8" (since this is the eighth volume).